metron-interface/metron-alerts/src/app/alerts/configure-table/configure-table.component.html

```diff
@@ -70,10 +70,26 @@
             <span class="text-uppercase"> {{ column.columnMetadata.type }} </span>
           </td>
           <td>
-            <span id="up-{{ column.columnMetadata.name }}" class="up" (click)="swapUp(i)" [ngClass]="{'disabled': i === 0}"></span>
+            <button id="up-{{ column.columnMetadata.name }}"
+                    class="up border-0 p-0"
+                    name="Move up"
+                    type="button"
+                    #moveColUpBtn
+                    aria-label="Move column up in display order"
+                    (mouseup)="swapUp(i, $event)"
+                    (keyup.enter)="swapUp(i, $event)"
```

Comment:
I had to change the events used here because, by default, browsers emit a click event when  enter is pressed while focused on a button. The default behavior would cause the up arrow button to lose focus, which differs in behavior when compared to the down arrow button. I very rarely find justification in overriding default browser behavior, but I strongly feel that it would be extremely annoying for keyboard users to have to backtab 4 times for every spot they want to move a column up in display order.

To fix this, I created two event bindings, one for mouseup and one for keyup.enter.

Reply:
`by default, browsers emit a click event when enter is pressed while focused on a button`

TIL